ironfish-cli/src/commands/wallet/multisig/create-signature-share.ts

```diff
@@ -28,6 +28,13 @@ export class CreateSignatureShareCommand extends IronfishCommand {
       description: 'The signing package for which the signature share will be created',
       required: false,
     }),
+    signerIdentity: Flags.string({
```

Comment:
will this always need to include all of the identities in the group? or all of the identities that were passed when creating the commitment?

Reply:
Only the ones that were passed when creating the commitment (i.e. the actual signers).

This argument can go away once we place this information into the signing package.